dynamodb_feature_store.js

```diff
@@ -26,12 +26,15 @@ function dynamoDBFeatureStoreInternal(tableName, options) {
     })
   );
   var dynamoDBClient = options.dynamoDBClient || new AWS.DynamoDB.DocumentClient(options.clientOptions);
+  var prefix = options.prefix || '';
 
-  this.getInternal = function(kind, key, cb) {
```

Comment:
I've removed all the `this`'s - I just feel like it's too error-prone in JS, it's so easy to forget to copy it to another variable.